routes/campgrounds.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+const router = express.Router();
+const Campground = require("../models/campground");
+const Comment = require("../models/comment");
+const middleware = require("../middleware");
+
+router.get("/campgrounds", function (req, res) {
```

Comment:
Instead of placing the callback function here, this is the time you’ll create Controllers where you place your logic there or atleast the feature’s facade.